scripts/upload-counterpart-agreements.py

```diff
@@ -72,8 +73,14 @@
 
     stage = arguments['<stage>']
     data_api_client = DataAPIClient(get_api_endpoint_from_stage(stage), get_auth_token('api', stage))
-    framework = data_api_client.get_framework(arguments['<framework_slug>'])["frameworks"]
+    framework_slug = arguments['<framework_slug>']
+    framework = data_api_client.get_framework(framework_slug)["frameworks"]
     document_directory = arguments['<documents_directory>']
+    content_path = arguments['<content_path>']
+    content_loader = ContentLoader(content_path)
+    content_loader.load_messages(framework_slug, ['e-signature'])
+    contract_title = content_loader.get_message(framework_slug, 'e-signature', 'framework_contract_title')
```

Comment:
What happens if we need to upload a counterpart agreement for an old framework that doesn't have this message? 🤔 Is there a default we can set?

Reply:
@lfdebrux Oh, good point, I'm not sure if we are still signing for G11 and back but I will set a default argument just in case.

Reply:
@lfdebrux addressed in c5b214a